src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -963,6 +963,7 @@ public static partial class JsonMetadataServices
         public static System.Text.Json.Serialization.JsonConverter<int> Int32Converter { get { throw null; } }
         public static System.Text.Json.Serialization.JsonConverter<long> Int64Converter { get { throw null; } }
         public static System.Text.Json.Serialization.JsonConverter<System.Text.Json.JsonElement> JsonElementConverter { get { throw null; } }
+        public static System.Text.Json.Serialization.JsonConverter<System.Text.Json.Nodes.JsonObject> JsonObjectConverter { get { throw null; } }
```

Comment:
Is this supposed to be part of this change? It seems unrelated...

Reply:
Per https://github.com/dotnet/runtime/pull/58912#discussion_r705797719, `JsonObject` can be used as ext. data. We need this to support that scenario (also for upcoming support for S.T.Json.Nodes types - https://github.com/dotnet/runtime/issues/45448#issuecomment-915449217).

Reply:
I must be missing something, but I'm not seeing where the generated code calls this. Can you point me to where it is used?

Reply:
The code that calls it is generated implicitly on this line: https://github.com/dotnet/runtime/blob/f4afd921e20fc7e2d7f17543784e8d453f5bf57e/src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs#L292

`{typeFriendlyName}Converter` resolves to `JsonObjectConverter`.